src/components/common/ActionModal.vue

```diff
@@ -365,10 +389,6 @@ export default {
         this.step = feeStep
       } catch ({ message }) {
         this.submissionError = `${this.submissionErrorPrefix}: ${message}.`
-
-        setTimeout(() => {
```

Comment:
Is that ok @jbibla? The hides pretty quick. I don't think hiding it is actually necessary